apps/admin/frontend/src/screens/manual_data_import_precinct_screen.tsx

```diff
@@ -328,7 +330,9 @@ export function ManualDataImportPrecinctScreen(): JSX.Element {
     existingPrecinctTally
   );
 
-  const writeInSummaryQuery = useWriteInSummaryQuery({ status: 'adjudicated' });
+  const writeInSummaryQuery = getWriteInSummary.useQuery({
+    status: 'adjudicated',
+  }) as UseQueryResult<Admin.WriteInSummaryEntryAdjudicated[]>;
```

Comment:
Previously there was an overload on the query that provided this. It's harder to overload a `grout` API so I'm just typecasting. We could just make a query specifically for `status: 'adjudicated`` however, as the frontend only uses the query with that status anyway.